lib/community/pages/create_post_page.dart

```diff
@@ -515,26 +516,31 @@ class _CreatePostPageState extends State<CreatePostPage> {
                               ],
                             ),
                             const SizedBox(height: 10),
-                            showPreview
-                                ? Container(
-                                    constraints: const BoxConstraints(minWidth: double.infinity),
-                                    decoration: BoxDecoration(border: Border.all(color: Colors.grey), borderRadius: BorderRadius.circular(10)),
-                                    padding: const EdgeInsets.all(12),
-                                    child: SingleChildScrollView(
-                                      child: CommonMarkdownBody(
-                                        body: _bodyTextController.text,
-                                        isComment: true,
-                                      ),
-                                    ),
-                                  )
-                                : MarkdownTextInputField(
-                                    controller: _bodyTextController,
-                                    focusNode: _bodyFocusNode,
-                                    label: l10n.postBody,
-                                    minLines: 8,
-                                    maxLines: null,
-                                    textStyle: theme.textTheme.bodyLarge,
+                            AnimatedCrossFade(
+                              firstChild: Padding(
+                                padding: const EdgeInsets.only(top: 8.0),
```

Comment:
We might be able to get rid of this `padding` and apply it to the `Container`'s margin instead!

Reply:
Good idea, done!